mkdocs.yml

```diff
@@ -71,6 +71,7 @@ pages:
         - 'rtcfifo': 'en/modules/rtcfifo.md'
         - 'rtcmem': 'en/modules/rtcmem.md'
         - 'rtctime': 'en/modules/rtctime.md'
+        - 'si7021' : 'en/modules/si7021'
```

Comment:
Here a `.md` is missing. Our RTD builds failed and thus https://nodemcu.readthedocs.io/en/latest/en/modules/si7021 return HTTP 404. I'll fix this.